### PR TITLE
Add location mapping july52022

### PR DESCRIPTION
**Pull request by msschroeder**
I have added updates to the XLSForm Content Documentation highlight the key changes to the syntax. There will be one more pull request as I could not edit the update file included as a part of the main and I forgot to include the update to the appearance section.

### CODE REVIEW

File: _sections/home-english.md [hunk 1]
```diff
@@ -233,8 +255,16 @@ If the CSV or XML files use different names for the choice `name` and `label`, a
...
+Caption: GeoJSON usage
 
-Note that this question type is generally the preferred way of building select questions from external data as it is the most versatile and works across applications. However, if your external data file consists of many thousands of lines, please test carefully whether the performance is satisfactory on the lowest spec device you intend to use. If it is too slow, consider using [External Selects](#external-selects) or [Dynamic selects from preloaded data](#dynamic-selects-from-pre-loaded-data) if your data collection application supports it. 
+Note that, this question type is generally the preferred way of building select questions from external data as it is the most versatile and works across applications. However, if your external data file consists of many thousands of lines, please test carefully whether the performance is satisfactory on the lowest spec device you intend to use. If the form is too slow for CSV or XML files, consider using [External Selects](#external-selects) or [Dynamic selects from preloaded data](#dynamic-selects-from-pre-loaded-data) if your data collection application supports it.
```
Comment (lognaturel) [inline _sections/home-english.md:267]: I think the original sentence better captured the intent! What it's saying is that if this way of attaching external data is slow in a particular context, there are other options.

File: _sections/home-english.md [hunk 2]
```diff
@@ -969,7 +999,8 @@ The **appearance** column allows you to change the appearance of questions in yo
...
 | table-list           | groups                      | An easier way to achieve the same appearance as above, apply this attribute to the entire group of questions (might slow down the form a bit).                                                                                  |
 | signature            | image                       | Allows you to trace your signature into your form (mobile clients only).                                                                                                                                                        |
-| draw                 | image                       | Allows you to sketch a drawing with your finger on the mobile device screen.                                                                                                                                                    |
+| draw                 | image                       | Allows you to sketch a drawing with your finger on the mobile device screen.     |
+| map, quick map       | select_one, select_one_from_file | Allows a user to select a choice from many locations on a map |
```
Comment (lognaturel) [inline _sections/home-english.md:1003]: locations -> features

**Review by lognaturel — CHANGES_REQUESTED** [review]
Couple suggestions to consider!

File: _data/choices.csv [hunk 3]
```diff
@@ -4,4 +4,5 @@ list name,A unique name for each set of choices,✔,✔,list_name,
 name,"ID (name) of the specific choice, will be saved to XML",✔,✔,,
 label::[language],"Choice label, will be displayed on screen. Allows adding a translation of choice labels, showing [language] in the interface",✔,✔,label::[language],
 media,,✔,✔,,optional
-[filter_category_name],Allows setting a specific parent category for choice filters (cascading questions),✔,✔,,"Column name is specified by user, e.g. ‘country’"
\ No newline at end of file
+[filter_category_name],Allows setting a specific parent category for choice filters (cascading questions),✔,✔,,"Column name is specified by user, e.g. ‘country’"
+geometry,Displays choices on map as locations,✔,✔,,
```
[the remark this answers is not in the document]
Comment (msschroeder) [reply _data/choices.csv]: I have updated this.

**Review by lognaturel — APPROVED** [review]

**Pull request merged**